libs/service/discord/src/handleDiscordMessaging.ts

```diff
@@ -1,5 +1,6 @@
 import { Api } from "@cennznet/api";
 import { InteractionWebhook } from "discord.js";
+import { Logger } from "winston";
```

Comment:
should we use the `getLogger` here?

Reply:
We are [here](https://github.com/cennznet/app-gov/blob/e32058deecd21e948631fcfd4437e7c1dd2fd578/libs/service/relayer/src/handleProposalActivityMessage.ts#L36) and passing it in - I assumed NX would complain about cyclical dependencies otherwise